src/pages/briefs/index.tsx

```diff
@@ -449,6 +478,40 @@ const Briefs = (): JSX.Element => {
             </div>
           ))}
         </div>
+        <Pagination
+          pageSize={3}
+          total={briefs.length}
+          onChange={(page: number, pageSize: number) =>
+            paginatedBriefs()?.paginate(page)
```

Comment:
this should be updated based on `paginatedBriefs`

like `paginatedBriefs(pageSize)?.paginate(page)`

Reply:
made it a variable that can be updated